website/docs/d/subnet.html.markdown

```diff
@@ -40,3 +40,4 @@ output "subnet_id" {
 * `route_table_id` - The ID of the Route Table associated with this subnet.
 * `ip_configurations` - The collection of IP Configurations with IPs within this subnet.
 * `service_endpoints` - A list of Service Endpoints within this subnet.
+* `disable_private_link_endpoint_network_policy_enforcement` - Enable or Disable network policies for a Private Link Endpoint in the subnet.
```

Comment:
I think this description needs to be updated?

Reply:
Fixed.